meta-xt-driver-domain/recipes-connectivity/xen-network/xen-network.bb

```diff
@@ -47,35 +49,41 @@ do_install() {
     install -m 0644 ${S}/bridge-up-notification.service ${D}${systemd_system_unitdir}
 
     install -d ${D}${sysconfdir}/systemd/network/
-    install -m 0644 ${S}/*.network ${D}${sysconfdir}/systemd/network
+    install -m 0644 ${S}/50-xenbr0.network ${D}${sysconfdir}/systemd/network
     install -m 0644 ${S}/*.netdev ${D}${sysconfdir}/systemd/network
 
+    install -m 0644 ${S}/template.network ${D}${sysconfdir}/systemd/network/${XT_DOMD_HW_ETH_NAME}.network
```

Comment:
IMO, it is  better to call it not `${XT_DOMD_HW_ETH_HANE}.network`, but just `external.network`, as it better reflects network name.